edit/buildozer.go

```diff
@@ -1236,12 +1233,14 @@ func Buildozer(opts *Options, args []string) int {
 	close(data)
 	records := []*apipb.Output_Record{}
 	hasErrors := false
+	fileModified := false
```

Comment:
for vars with a default value, 

var fileModified bool

is the internal convention

Reply:
Done